cluster-autoscaler/config/autoscaling_options.go

```diff
@@ -46,6 +46,16 @@ type NodeGroupAutoscalingOptions struct {
 	ScaleDownUnreadyTime time.Duration
 }
 
+// GCESpecificOptions contain autoscaling options specific to GCE cloud provider.
+type GCESpecificOptions struct {
+	// ConcurrentGceRefreshes is the maximum number of concurrently refreshed instance groups or instance templates.
+	ConcurrentGceRefreshes int
```

Comment:
All these variables should not have `Gce` in them, as they are already defined in `GCESpecificOptions`.

Reply:
makes sense, done.